core-services/prow/02_config/stolostron/cluster-proxy-addon/_pluginconfig.yaml

```diff
@@ -0,0 +1,10 @@
+plugins:
```

Comment:
Not sure if an `OWNERS` file is required?

Reply:
I thought OWNERS are auto-genrated by prow, I will follow up, if it's not created, I will create it.